screens/AddPostScreen.js

```diff
@@ -0,0 +1,184 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Text,
+  View,
+  TouchableOpacity,
+  Image,
+  KeyboardAvoidingView,
+  Platform,
+  TouchableWithoutFeedback,
+  Keyboard,
+} from 'react-native';
+import { Camera } from 'expo-camera';
+import { t } from 'react-native-tailwindcss';
+import { Headline, TextInput } from 'react-native-paper';
+import StyledButton from '../components/StyledButton';
+
+export default function AddPostScreen() {
+  const [hasPermission, setHasPermission] = useState(null);
+  const [cameraRef, setCameraRef] = useState(null);
+  const [type, setType] = useState(Camera.Constants.Type.back);
+  const [imageUri, setImageUri] = useState('');
+  const [searchBarVisible, setSearchBarVisibility] = useState(false);
+  const [discountPrice, setDiscountPrice] = useState(null);
+  const [regularPrice, setRegularPrice] = useState(null);
+
+  useEffect(() => {
+    (async () => {
+      const { status } = await Camera.requestPermissionsAsync();
+      setHasPermission(status === 'granted');
+    })();
+  }, []);
+
+  if (hasPermission === null) {
+    return <View />;
+  }
+  if (hasPermission === false) {
+    return <Text>No access to camera</Text>;
+  }
+
+  const takePicture = async () => {
+    if (cameraRef) {
+      const photo = await cameraRef.takePictureAsync();
+      // photo uri to store in the db
+      // console.log('photo', photo.uri);
+      return setImageUri(photo.uri);
+    }
+    return setImageUri('');
+  };
+
+  function cancelPhoto() {
+    setImageUri('');
+    setSearchBarVisibility(false);
+  }
+
+  return (
+    <View style={[t.flex1]}>
+      {!imageUri ? (
+        // Camera Screen
+        <Camera
+          style={[t.flex1]}
+          type={type}
+          ref={(ref) => {
+            setCameraRef(ref);
+          }}
+        >
+          <View style={[t.flex1, t.bgTransparent, t.justifyEnd]}>
+            <TouchableOpacity
+              style={[t.selfEnd, t.mR16]}
+              onPress={() => {
+                setType(
+                  type === Camera.Constants.Type.back
+                    ? Camera.Constants.Type.front
+                    : Camera.Constants.Type.back
+                );
+              }}
+            >
+              <Text style={{ fontSize: 18, marginBottom: 10, color: 'white' }}> Flip </Text>
+            </TouchableOpacity>
+            <TouchableOpacity style={[t.selfCenter]} onPress={takePicture}>
+              <View
+                style={{
+                  borderWidth: 2,
+                  borderRadius: '50%',
```

Comment:
![Screenshot from 2020-06-23 08-08-59](https://user-images.githubusercontent.com/6239834/85402029-fb709480-b528-11ea-9b09-40020c574b98.png)

might be related to this too: https://github.com/facebook/react-native/issues/23219

Reply:
It's an issue related to Android. I fixed with break it down the borderRadius: https://github.com/react-native-community/react-native-video/issues/685.